### PR TITLE
Detect local or global vendor/autoload.php

### PR DESCRIPTION
**Pull request by vluzrmos**
That is for use a `git clone` studio.

### CODE REVIEW

**Comment by franzliedke** [comment]
Thanks! =)

Can you explain the exact use case, please?

File: bin/studio [hunk 1]
```diff
@@ -1,7 +1,17 @@
 #!/usr/bin/env php
 <?php
 
-require __DIR__.'/../../../autoload.php';
+$autoloaders = array(
+	__DIR__.'/../vendor/autoload.php',
+	__DIR__.'/../../../autoload.php'
+);
```
Comment (franzliedke) [inline bin/studio]: You can use the short array syntax here. :)

**Comment by vluzrmos** [comment]
That is in case of you can do a clone of the repo or `composer create-project`, and use that "/bin/studio" as studio path.

**Comment by franzliedke** [comment]
Thank you! Just came across needing this, too ;)

**Pull request merged**